exec/java-exec/src/test/java/org/apache/drill/exec/store/sys/TestSystemTable.java

```diff
@@ -61,6 +61,12 @@ public void optionsOrderBy() throws Exception {
     test("select * from sys.options order by name");
   }
 
+  @Test
+  public void testSerDe() throws Exception {
+    PlanTestBase.testPhysicalPlanExecutionBasedOnQuery("select * from sys.threads");
+    PlanTestBase.testPhysicalPlanExecutionBasedOnQuery("select * from sys.connections limit 10");
```

Comment:
This test is sort of OK. (It converts the plan to a physical plan, then executes it.) But, we don't check the results.

A better SerDe test is to convert the object to JSON, then convert back and compare the two objects. (I have such a test I added for the column metadata serialization for which I need to issue a PR after the release.)

Reply:
Added test to verify ser / de plus results.